Sources/RegexBenchmark/Suite/Backtracking.swift

```diff
@@ -0,0 +1,45 @@
+import _StringProcessing
+import RegexBuilder
+import Foundation
+
+// Tests that involve heavy backtracking
+
+extension BenchmarkRunner {
+  mutating func addBacktracking() {
+    let r = "^ +A"
+    let s = String(repeating: " ", count: 10000)
+
+    let basicBacktrack = Benchmark(
+      name: "BasicBacktrack",
+      regex: try! Regex(r),
+      ty: .allMatches,
```

Comment:
Go ahead and give the `ty` argument label a full name. That is ok as a argument name, but the label should clarify the use site.